docs/pda/class-pda.md

```diff
@@ -5,6 +5,20 @@
 The `PDA` class is an abstract base class from which all pushdown automata
 inherit. It can be found under `automata/pda/pda.py`.
 
+The `PDA` class has the following abstract methods:
+
+## PDA.show_diagram(self, input_str = None, path = None):
+
+Constructs and returns a pygraphviz `AGraph` corresponding to this PAD. If `input_str` is
```

Comment:
@Vipul-Cariappa Typo here. `PAD` should be `PDA`.

```suggestion
Constructs and returns a pygraphviz `AGraph` corresponding to this PDA. If `input_str` is
```

Reply:
I have fixed the typo.